glmnet/tests/test_linear.py

```diff
@@ -102,6 +102,15 @@ def test_alphas(self):
             m = m.fit(x, y)
             self.check_r2_score(y, m.predict(x), 0.90, alpha=alpha)
 
+    def test_coef_limits(self):
+            x, y = self.inputs[0]
+            lower_limits = 0
+            upper_limits = np.repeat(1,x.shape[1])
```

Comment:
Since this test fails to catch the bug in the bounds checking, how about you try re-writing it with lower limits of `np.repeat(-1, x.shape[1])` and upper limits of 0.